bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java

```diff
@@ -2050,6 +2059,53 @@ protected void readLedgerIndexEntries(long ledgerId) throws IOException {
     }
 
     /**
+     * Read ledger index entries
+     * 
+     * @param ledgerId
+     * @return
+     * @throws IOException if failed to init KeyValueStorageRocksDB
+     */
+	protected int readLedgerIndexEntriesFromDbLedgerStorage(long ledgerId) throws IOException {
```

Comment:
Minor thing: Since this is specific to DbLedgerStorage, should we have the code in that class too?